src/pages/components/combobox.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Combobox
+description: Combobox is an input plus a list box that appears beneath it. It is used for search and filtering.
+package: '@zendeskgarden/react-dropdowns'
+components:
+  - dropdowns/src/elements/Combobox/Combobox.tsx
+  - dropdowns/src/elements/Dropdown/Dropdown.tsx
+  - dropdowns/src/elements/Fields/Field.tsx
+  - dropdowns/src/elements/Fields/Hint.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - dropdowns/src/elements/Fields/Label.tsx
+  - dropdowns/src/elements/Menu/Menu.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To filter down a list of options</li>
+      <li>To quickly find a known option</li>
+      <li>As a search field</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For a searchable dropdown, use [Autocomplete](/components/autocomplete) instead
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>
+          To make more than one selection, use [Multiselect](/components/multiselect) instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+As the user types, options appear in the list box that match what the user is typing.
+
+import ComboboxDefault from '../../examples/dropdowns/combobox/Default.tsx';
+import ComboboxDefaultCode from '!!raw-loader!../../examples/dropdowns/combobox/Default.tsx';
+
+<CodeExample code={ComboboxDefaultCode}>
+  <ComboboxDefault />
+</CodeExample>
+
+## Customization
+
+### Size
+
+Combobox can be displayed in either default or compact size.
+
+import ComboboxSize from '../../examples/dropdowns/combobox/Size.tsx';
+import ComboboxSizeCode from '!!raw-loader!../../examples/dropdowns/combobox/Size.tsx';
+
+<CodeExample code={ComboboxSizeCode}>
+  <ComboboxSize />
+</CodeExample>
+
+### Hidden Label
+
+The label can be hidden as needed.
+
+import ComboboxHidden from '../../examples/dropdowns/combobox/Hidden.tsx';
+import ComboboxHiddenCode from '!!raw-loader!../../examples/dropdowns/combobox/Hidden.tsx';
+
+<CodeExample code={ComboboxHiddenCode}>
+  <ComboboxHidden />
+</CodeExample>
+
+### Selection Logic
+
+The combobox can be customized to include selection logic that highlights the first relevant
+result as the user types, allowing the user to hit enter to fully select the item. If this logic
```

Comment:
I'm noticing this now: "hit enter" sounds really colloquial in (American?) English. 
Let's change this to "...allowing the user to `press Enter to select the item`". 

I also want to remove "fully" because I don't think it's adding anything to the sentence in terms of understanding

Reply:
Updated!